docs/resources/permissions.md

```diff
@@ -621,7 +621,7 @@ resource "databricks_permissions" "token_usage" {
 
 ## SQL warehouse usage
 
-[SQL warehouses](https://docs.databricks.com/sql/user/security/access-control/sql-endpoint-acl.html) have three possible permissions: `IS_OWNER`, `CAN_USE` and `CAN_MANAGE`:
+[SQL warehouses](https://docs.databricks.com/sql/user/security/access-control/sql-endpoint-acl.html) have four possible permissions: `IS_OWNER`, `CAN_USE`, `CAN_MONITOR` and `CAN_MANAGE`:
```

Comment:
Would it make sense to order them consistent with the privileges? CAN_USE < CAN_MONITOR < CAN_MANAGE <= IS_OWNER

therefore 

```
have four possible permissions: `CAN_USE`, `CAN_MONITOR`, `CAN_MANAGE` and `IS_OWNER`:
```